src/main/java/bio/terra/service/filedata/FSFileInfo.java

```diff
@@ -88,4 +89,15 @@ public String toString() {
         .append("bucketResourceId", bucketResourceId)
         .toString();
   }
+
+  public static FSFileInfo getBaseInstance(String fileId, String resourceId) {
```

Comment:
I think a name like `getTestInstance` or `getFaultTestInstance` would be a bit clearer, since that is the only place it's used

Reply:
Great point!

Reply:
I originally misinterpreted the goal of this object